HW03-VaughtK/hw03_VaughtK.py

```diff
@@ -0,0 +1,182 @@
+
+# TODO Do not edit this file directly!
+# TODO Instead, create a new file called hw03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Kai Vaught
+# Username: VaughtK
+#
+# Assignment: Homework 3
+# Purpose: To learn how branches work.
+# Google Doc Link: https://docs.google.com/document/d/1OvsmR456y61n3UzwBlIlVE13IBz0tF39Vyv10F24ZGI/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle
+
+
+def house_turtle():
+    import turtle
+
+    t = turtle.Turtle()
+
+    t.setpos(0,-100)
+    t.color("green")
+    t.width(4)
+    t.speed(0)  # 1:slowest, 3:slow, 5:normal, 10:fast, 0:fastest
+    t.screen.bgcolor("black")
+    t.left(60)
+    t.forward(90)
+    t.left(30)
+    t.forward(90)
+    t.left(85)
+    t.forward(90)
+    t.left(55)
+
+    t.forward(90)
+    t.left(35)
+    t.forward(90)
+    t.forward(-90)
+    t.left(90)
+    t.forward(90)
+    t.left(55)
+    t.forward(90)
+    t.forward(-90)
+    t.left(35)
+    t.forward(-90)
+    t.left(-90)
+    t.forward(120)
+    t.left(-100)
+    t.forward(-80)
+    t.left(-80)
+    t.forward(80)
+    t.left(70)
+    t.forward(90)
+    t.right(70)
+    t.forward(40)
+    t.right(90)
+    t.forward(20)
+    t.left(90)
+    t.forward(10)
+    t.left(90)
+    t.forward(25)
+    t.right(90)
+    t.forward(60)
+    t.forward(-210)
+    t.right(90)
+    t.forward(20)
+    t.forward(-20)
+    t.right(-90)
+    t.forward(215)
+    t.right(90)
+    t.forward(30)
+    t.right(90)
+    t.penup()
+    t.forward(15)
+    t.pendown()
+    t.forward(30)
+    t.right(90)
+    t.forward(20)
+    t.right(90)
+    t.forward(30)
+    t.right(90)
+    t.forward(20)
+    t.right(90)
+    t.penup()
+    t.setpos(150,150)
+    t.pendown()
+    t.pensize(25)
+    t.pencolor("white")
+
+    t.right(30)
+    t.forward(5)
+    t.right(30)
+    t.forward(5)
+    t.right(30)
+    t.forward(5)
+    t.right(30)
+    t.forward(5)
+    t.right(30)
+    t.forward(5)
+    t.right(30)
+    t.forward(5)
+    t.right(30)
+    t.forward(5)
+    t.right(30)
+    t.forward(5)
+    t.right(30)
+    t.forward(5)
+    t.right(30)
+    t.forward(5)
+    t.right(30)
+    t.forward(5)
+    t.right(30)
+    t.forward(5)
+
+    t.pensize(5)
+
+
+    t.penup()
+    t.setpos(-150,150)
+    t.pendown()
+    t.forward(5)
+
+    t.penup()
+    t.setpos(-130,170)
+    t.pendown()
+    t.forward(5)
+
+    t.penup()
+    t.setpos(10,210)
+    t.pendown()
+    t.forward(5)
+
+    t.penup()
+    t.setpos(-210,130)
+    t.pendown()
+    t.forward(5)
+
+    t.penup()
+    t.setpos(220,190)
+    t.pendown()
+    t.forward(5)
+
+    t.penup()
+    t.setpos(210,130)
+    t.pendown()
+    t.forward(5)
```

Comment:
A loop could probably streamline these repeating lines of code. You'll have to figure out how to get the numbers into the loop... maybe a list? I can help you do this, since we haven't learned lists yet!